doc/tech/glossary.md

```diff
@@ -106,8 +107,33 @@ Terms not defined by Kubernetes:
 - **CaaS**:	Container-as-a-Service. A complete set of technologies to enable the management of containerised software, including a Kubernetes cluster, container networking, storage, routing, service mesh, etc.
 - **CaaS Manager**:	A management plane function that manages the lifecycle (instantiation, scaling, healing, etc.) of one or more CaaS instances, including communication with VIM for master/node lifecycle management.
 
-<a name="1.5"></a>
-## 1.5 Other Referenced Terminology
+
+<a name="1.6"></a>
+## 1.5 OpenStack Related Terminology
+
+>_*Note:*_ The official [OpenStack Glossary]( https://docs.openstack.org/image-guide/common/glossary.html) is an extensive list of OpenStack-related concepts. Some additional terms used in the Reference Architecture RA-1 or used to relate RA-1 terms with terms defined elsewhere.
+
+**Core (physical)** is an independent computer processing unit (pcpu) that can independently execute CPU instructions but is integrated with other cores on a multiprocessor chip (integrated circuit die). Please note that the multiprocessor chip is also referred to as a CPU that is placed in a socket of a computer motherboard.
+
+**Flavor Capabilities** such as (CPU Pinning, NUMA< huge pages): CNTT NFVI Profile
+
+**Flavor Geometry** Flavor sizing such as number of vCPUs, RAM, disk, etc.
+
+**Fluentd** is an open-source data collector that both collects data, transforms it and delivers it for consumption.
+
```

Comment:
**Fluentd ** is an open-source data collector for unified logging layer, which allows you to unify data collection and consumption for better use and understanding of data, it is CNCF graduated project "https://www.fluentd.org/"

Reply:
@ASawwaf Thanks. Changed to:
**[Fluentd](https://www.fluentd.org/)**, a CNCF graduated project, is an open-source data collector for unified logging layer, which allows data collection and consumption for better use and understanding of data.